File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ContemporaryDiningTable.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ContemporaryDiningTable implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a contemporary dining table\n";
    }
}

File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ContemporaryDiningTable.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ContemporaryDiningTable : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a contemporary dining table");
        }
    }
}
